main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 				PerfBufferSize:        c.Int("perf-buffer-size"),
 				PidsToTrace:           c.IntSlice("pid"),
 				BlobPerfBufferSize:    c.Int("blob-perf-buffer-size"),
-				OutputPath:            c.String("output-path"),
+				OutputDir:             c.String("output-dir"),
```

Comment:
Why changing path to dir?
Today we save to a dir, maybe tomorrow to tar.gz, who knows?

Reply:
ok